include/odp/api/spec/atomic.h

```diff
@@ -524,6 +596,28 @@ int odp_atomic_cas_rel_u32(odp_atomic_u32_t *atom, uint32_t *old_val,
 int odp_atomic_cas_acq_rel_u32(odp_atomic_u32_t *atom, uint32_t *old_val,
 			       uint32_t new_val);
 
+/**
+ * Set bits in atomic uint32 variable using RELEASE memory ordering
+ *
+ * Otherwise identical to odp_atomic_bit_set_u32() but ensures RELEASE memory
+ * ordering.
+ *
+ * @param atom    Pointer to atomic variable
+ * @param bits    Bits to set
+ */
+void odp_atomic_bit_set_rel_u32(odp_atomic_u32_t *atom, uint32_t bits);
```

Comment:
Now that the release variants are here, maybe an acq-rel variant of the fetch_set and fetch_clr functions would be useful too.

Reply:
There are no non-relaxed fetch versions of any kind, so seems that they are not that useful.

Reply:
They can anyway be in a separate commit and PR.